tutorials/math/matrices_and_transforms.rst

```diff
@@ -308,36 +308,25 @@ as well as :ref:`Spatial.to_local() <class_Spatial_method_to_local>` and :ref:`S
 Scale
 ~~~~~
 
-A matrix can be scaled too. Scaling will multiply the basis vectors by a
-vector (X vector by x component of the scale, Y vector by y component of
-the scale). It will leave the origin alone:
+A matrix can be scaled too. Scaling will multiply the X and Y vectors of the
+matrix by the ``x`` and ``y`` components of another vector (``X * x``, ``Y * y``).
+It will leave the origin alone:
 
 .. tabs::
  .. code-tab:: gdscript GDScript
 
-    # Make the basis twice its size.
+    # Scale the transform's vectors to twice their size
     var m = Transform2D()
     m = m.scaled( Vector2(2,2) )
 
  .. code-tab:: csharp
 
-    // Make the basis twice its size.
+    // Scale the transform's vectors to twice their size
     var m = Transform2D.Identity;
     m = m.Scaled(new Vector2(2, 2));
 
 .. image:: img/tutomat15.png
 
-These kind of operations in matrices are accumulative. It means every
-one starts relative to the previous one. For those who have been living
```

Comment:
I am not overly attached to the Santa joke. However, it is important to note the accumulative nature of matrix operations. Also, many people learn programming with a turtle graphics API, so a reference to a turtle API is useful, it doesn't need to carry the same tone if you find it inappropriate.

Reply:
> However, it is important to note the accumulative nature of matrix operations.

The language in this sentence was a bit confusing, but I get what it was trying to convey now. I'll add a note about the cumulative properties.

> Also, many people learn programming with a turtle graphics API, so a reference to a turtle API is useful, it doesn't need to carry the same tone if you find it inappropriate.

As a new user I had no idea what this was talking about, or what a turtle API is. Searching online doesn't give me much more of a clue. The image was a star, which didn't have anything to do with turtles.

Either I'm too young to get the reference or it needs to be explained a bit better :)

Reply:
The image is indeed outdated and only recognizable to older people and should be removed. But turtle graphics APIs are still around. I believe scratch uses one, Processing can use one. https://en.wikipedia.org/wiki/Turtle_graphics

Even if users are not already familiar, playing around with a turtle is a great way to get an intuitive understanding of matrices.

Reply:
Ahh ok. Perhaps then the example should be updated to pose a hypothetical scenario where you want to tell a turtle how to move, and you talk to it in terms of matricies.

Then say that this was a way people manipulated matricies in some systems.

Reply:
I don't think that is necessary. The idea is to capture the subset of readers who already understand turtle graphics and make the connection. I would just reference it like "for those of you that have used turtle graphics APIs before, this may seem familiar as turtle graphics use a matrix internally" 

Of course, use better language. You can also link to the wiki for users that are curious.

Reply:
I see, ok. I'll try to make that happen :)

Reply:
Bump :)